modules/imap_folders/modules.php

```diff
@@ -212,30 +212,34 @@ public function process() {
                         if ($linked_mailboxes && in_array($old_folder, $linked_mailboxes)) {
                             require_once VENDOR_PATH.'autoload.php';
                             $sieve_options = explode(':', $imap_account['sieve_config_host']);
```

Comment:
Shouldn't we parse this configuration with the new parse_sieve_config_host function?